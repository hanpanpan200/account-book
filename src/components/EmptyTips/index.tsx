import React from 'react';
import emptyIcon from 'assets/icons/empty.svg';

import styles from './index.module.scss';

interface Props {
  guidance: string
}

const EmptyTips: React.FC<Props> = ({ guidance }) => {
  return (
    <div className={styles.container}>
      <img alt='emptyIcon' src={emptyIcon}/>
      <div>{guidance}</div>
    </div>
  )
};

export default EmptyTips;
