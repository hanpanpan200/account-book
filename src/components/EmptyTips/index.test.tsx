import React from 'react';
import { render } from '@testing-library/react';
import EmptyTips from './index';

describe('EmptyTips', () => {
  it('should render correctly', () => {
    const { container } = render(<EmptyTips guidance='暂时无记录' />);
    expect(container).toMatchSnapshot();
  });
});
