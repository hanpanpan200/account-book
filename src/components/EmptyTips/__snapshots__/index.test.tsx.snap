// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EmptyTips should render correctly 1`] = `
<div>
  <div
    class="container"
  >
    <img
      alt="emptyIcon"
      src="empty.svg"
    />
    <div>
      暂时无记录
    </div>
  </div>
</div>
`;
